intel/intel_bufmgr_fake.c

```diff
@@ -1185,8 +1187,8 @@ static int
 
 	assert(bo_fake->block);
 	assert(bo_fake->block->bo == &bo_fake->bo);
-
-	bo->offset = bo_fake->block->mem->ofs;
+        if (!bo_fake->block)
```

Comment:
Alignment not proper ?
Also, isn't the assert above already verifying this condition ? Did you mean to check `bo_fake->block->mem` ?